data/starts.txt

```diff
@@ -0,0 +1,203 @@
+	`	You sign your name one final time:`
+	name
+	`	The elevator is so well-tuned that you do not even realize it is moving until it has deposited you back in the lobby. But as you leave the bank, you are smiling. This crazy adventure suddenly feels real to you. You are going to do it. You are finally going to get off this planet.`
+	`	Compared to the bank, you feel much more at home in the shipyard, walking among the rusted out hulks and newer ships that gleam in the sunlight. You smell grease and dirt and rocket fuel; wonderful smells. There are three ship models within your price range. Which one you choose will determine your future...`
+
+
+
+start "hai space"
+	name "New Hai"
+	description `You grew up on Greenwater, a world with a very rich and mixed culture. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. In order to reach that goal, you worked hard at a travel company, recommending mostly locations in human space.`
+	description `	Now, as you enter the shipyard, you feel like this is the start of a new life as a captain.`
+	thumbnail "scene/hai start"
+	system "Fah Soom"
+	planet "Greenwater"
+	date 16 11 3013
+	to display
+		or
+			has "main plot completed"
+			has "Ask the Hai about the Unfettered"
+	on display
+		description "What if you had not grown up on New Boston? What if you had waited longer before taking to the skies?"
+		thumbnail ""
+		date "3020"
+		planet "Slightly too warm"
+		credits "over a million"
+	to reveal
+		has "main plot completed"
+		has "Ask the Hai about the Unfettered"
+	on reveal
+		name "Undiscovered Hai Start"
+		description "There could be interesting possibilities up north, once all crises are resolved."
+		description "	You could have started a different way, went a different path."
+		system "Somewhere far north"
+		planet "Mixed culture"
+		date "Same old one"
+		credits 1.431M
+		debt 1.431M
+	to unlock
+		has "main plot completed"
+		has "event: hai-human resolution announced"
+	conversation "hai intro"
+	account
+		credits 1431000
+		score 400
+		mortgage Mortgage
+			principal 1431000
+			interest 0.003
+			term 730
+	set "species: human"
+	set "hai space start"
+	# need to set it here to not have it trigger alongside the invisible mission
+	set "First Contact: Hai: offered"
+	set "Intro [0]: offered"
+
+
+
+conversation "hai intro"
+	scene "scene/hai shipyard"
+	`You've never been to the shipyard before, and going to the North Pole already feels like you're traveling to another world. The pictures and videos you've seen of all these ships before simply don't do them justice; seeing them for real is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look. You begin to wonder if you are ready to take the big leap.`
```

Comment:
Since you're from the equator, you'll probably not have seen so many Hai in one place. The player should probably comment on that, unless they've been to the North Pole before.

Also, where was this travel agency, when did you work for it? Did you need to work there to save up for a license? Was it up in the North Pole where the shipyard/spaceport is? It doesn't make too much sense if you "feel you're traveling to another world" if you work at a travel agency in the North Pole. If you don't mention it here, it'll just be confusing later on in the Project Hawking and Human to Hai Space missions.
```suggestion
	`You've never been to a shipyard before, and going to the North Pole already makes you feel as if you're on another world. Though you've met some Hai before, you've never seen so many of them in one place; the pace of this Hai city far outstrips the humid human town on the equator where you grew up.`
	`	The pictures and videos you've seen of all these ships before simply don't do them justice; seeing them for real is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look. You begin to wonder if you are ready to take the big leap.`
```

Reply:
the agency wasnt on the pole, it was in the same mostly human city as that other dude you bring back to hai space in a mission

Reply:
What do you think of the suggestion above, though?

Reply:
It seems as if they haven't been to a Hai city, but otherwise, sure. I can change it.

Reply:
```suggestion
	`You've never been to the shipyard before, and going to the North Pole already makes you feel as if you're on another world. Though you've met some Hai in the past, you've never been among so many of them in one place; the pace of this Hai city far outstrips the muggy human town on the equator where you grew up.`
	`	The pictures and videos you've seen of all these ships before simply don't do them justice; seeing them for real is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look. You begin to wonder if you are ready to take the big leap.`
```